include/vkd3d.h

```diff
@@ -165,6 +165,9 @@ HRESULT vkd3d_create_root_signature_deserializer(const void *data, SIZE_T data_s
 
 VkFormat vkd3d_get_vk_format(DXGI_FORMAT format);
 
+uint32_t vkd3d_get_priority_adjust(VkDeviceSize size);
```

Comment:
The appropriate place to add internal functions is vkd3d_private.h. vkd3d.h is mostly vestigal at this point.